d2l-localize-behavior.js

```diff
@@ -130,12 +138,48 @@ D2L.PolymerBehaviors.LocalizeBehaviorImpl = {
 			return parseTime(val);
 		};
 	},
-	_computeLanguage: function(resources, lang, fallback) {
+	__computePossibleLanguages: function(language, fallbackLanguage) {
+		const langs = [ language, fallbackLanguage ]
+			.filter(e => e)
+			.map(e => [ e.toLowerCase(), e.split('-')[0] ])
+			.flat();
+
+		return [ ...new Set([ ...langs, 'en-us', 'en' ]) ];
+	},
+	_computeLanguage: function(resources, lang, fallback, resolvedLang) {
+		if (this.localizeConfig.importFunc) return resolvedLang;
```

Comment:
So based on what I am seeing, this can merge anytime as a minor bump and everywhere will continue to work as is, and we can slowly move each place over to use the "dynamic" functionality by setting the `localizeConfig`?